cmd/egnkey/README.md

```diff
@@ -0,0 +1,19 @@
+## egnkey
+
+### How to build
+Navigate to [egnkey](../egnkey/) directory and run
```

Comment:
We should have instructions for how to install via go install instead. This way we can use this binary anywhere

Reply:
yea. agreed. will add it in next PR. I need to add a funding utility too for DA

Reply:
added `go install` instructions